registries/adapters/adapter.go

```diff
@@ -20,14 +20,15 @@ import (
 	"bytes"
 	b64 "encoding/base64"
 	"encoding/json"
-	"io/ioutil"
-	"net/http"
 	"net/url"
 	"strconv"
 
+	"io/ioutil"
+	"net/http"
+
 	"github.com/automationbroker/bundle-lib/bundle"
 	log "github.com/sirupsen/logrus"
-	yaml "gopkg.in/yaml.v1"
+	"gopkg.in/yaml.v1"
```

Comment:
Any reason you removed the prefix? It works either way, but the first one wasn't necessarily wrong.

Reply:
No again I think this was the new editor